.github/workflows/ci-java.yml

```
@@ -0,0 +1,76 @@
+          mvn versions:set -DnewVersion=${{ steps.schema.outputs.schema_version }} -DgenerateBackupPoms=false
+
+      - name: Build with Maven
+        run: mvn clean package
+
+      - name: Publish to GitHub Packages
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: mvn deploy --batch-mode
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)